### ISSUE TITLE
Improve keyboard focus of modals

### ISSUE THREAD

**Issue by bmtymrak** [bmtymrak/carrots-budget#15]
The keyboard focus when opening any modal should be on the first input element of the form.  Tab focus should also be trapped to the modal.
